atd-vze/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16-bullseye
+RUN apt-get update
+RUN apt-get install -y vim aptitude magic-wormhole
+WORKDIR /root/
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+RUN npm install -g npm-check-updates
+WORKDIR /root/atd-vze
+#COPY . /root/atd-vze
+#RUN npm install
```

Comment:
are these commented out lines left in here so we have a different way to install?

Reply:
They are there as a reminder that we have an option of bundling a snapshot of the application into the docker image, and then being able to bind-mount the actual local copy of the files over that. This would be something we might do if we ever were to want to have more control over the web-server that serves them up, which is currently netlify. Like if we wanted to move it to ECS, for example.

I think thought that they are not helpful right now in there, and doubly so without comments at least explaining what they are. I am going to remove them - thank you @chiaberry.